dataplane/linux/bpf_ep_mgr.go

```diff
@@ -111,6 +111,10 @@ func newBPFEndpointManager(
 		dsrEnabled:          dsrEnabled,
 		ipSetMap:            ipSetMap,
 		stateMap:            stateMap,
+		mapCleanupRunner: ratelimited.NewRunner(10*time.Second, func(ctx context.Context) {
```

Comment:
I wonder if the timeout should be configurable. Or at least defined somewhere at the top as a constant?

Reply:
Moved to a constant, don't think it's worth the extra work for a config flag.